recipes/tetgen/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "tetgen" %}
+{% set version = "1.5.0" %}
+{% set sha256 = "4d114861d5ef2063afd06ef38885ec46822e90e7b4ea38c864f76493451f9cf3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://wias-berlin.de/software/tetgen/1.5/src/tetgen{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  features:
+    - vc9  # [win and py27]
+    - vc14  # [win and (py35 or py36)]
```

Comment:
Even though we are not building `vc10` (Python 3.4) in conda-forge it is nice to leave the `vc 10  # [win and py34]` here so others can use the recipe.